### ISSUE TITLE
Multi-language / Unicode support for saved presets

### ISSUE THREAD

**Issue by nalegke** [Stekeblad/Stekeblads-Video-Uploader#15]
Hi Stekeblad, could you add the multi-language support for saved presets?
I have tried to use Asian and Cyrillic characters for custom presets but they all turn into the question marks after restarting the program.
This applies to Stekeblads_Video_Uploader-1.4.0.jar / JRE 8 update 251.

**Comment by Stekeblad** [comment]
Hi nalegke,
I thought that most types of characters worked, but apparently not. I have looked into this and found a way to support most types of characters.

I have tested with Arabic, Russian, Japanese and simplified Chinese and it now loads correctly after a restart. I have also tried with random Unicode characters and if they look right in the program when you enter them, they load correctly as well. If some symbols still do not look right in the next update when you enter them it may be related to missing support in the current font, and if so highly likely others will have problem viewing them as well, and therefore it may be better to not use the broken symbols.

The update is available right now on the releases page or directly using the built in update checker
